src/Category/Tests/Api/GraphQl/CategoryTreeTest.php

```diff
@@ -180,7 +179,7 @@ protected function getCategoryTree(?string $catalogCode = null, ?string $localiz
                       }
                     }
                 GQL,
-                $this->getUser(Role::ROLE_ADMIN),
+                null,
```

Comment:
Shouldn't we also test that providing a user still grants us access to the endpoint ?

Reply:
Not useful, because the authenticated users have an access to the pages of "anonymous users".